statdump_to_json.pl

```diff
@@ -3,7 +3,7 @@
 use strict;
 use JSON;
 
-use lib "/lupus/ngi/production/latest/sw/arteria/siswrap_venv/deps/sisyphus/sisyphus-latest/lib";
+use lib "/vulpes/ngi/production/latest/sw/arteria/siswrap_venv/deps/sisyphus/sisyphus-latest/lib";
```

Comment:
This path doesn't exist. It seems to be a sisyphus thing... can we just remove this script?

Reply:
yes, that should be ok